### PR TITLE
Added README for andino_bringup

### LINKED ISSUE #146: Add documentation for the andino_bringup package
### Summary

`andino_bringup` package provides some convenient launch files for launching the robot stack as well some other external tools.

### Definition of Done.

 - [ ] Provide a README file with general information
 - [ ] Describe in the README file how to use each launch file.

### PR DESCRIPTION
**Pull request by JesusSilvaUtrera**
# 🎉 New feature

Closes #146 

## Summary
I have added the README.md file to the `andino_bringup` package, explaining the contents of the package.

## Test it
You can check the file and read it to see if it is complete.

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [x] Added example and/or tutorial
- [x] Updated documentation (as needed)

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: andino_bringup/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+It includes:
+
+- `andino_description.launch.py`: file to load the 3D model of the robot from the URDF.
+- `andino_control.launch.py`: file for the ROS2 control nodes and drivers.
+- `camera.launch.py`: file to execute the camera drivers, using `v4l2_camera` package. If not necessary, it can be disabled using the `include_camera` parameter.
+- `rplidar.launch.py`: file to execute the drivers of the RP 2D LiDAR attached to the robot. If not necessary, it can be disabled using the `include_rplidar` parameter.
+
+### Other launch files included
+
+These launch files are included just in case we want to command the robot in different ways, or visualize the information:
```
Comment (jballoffet) [inline andino_bringup/README.md:21]: nit: "... in case you want ..."

File: andino_bringup/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+- `rosbag_record.launch.py`: executes the `ros2 bag record` command to store a '.bag' file with the specified topics. If no topics are specified, it will record all of them.
+- `rviz.launch.py`: executes the RViz visualization tool with the 'andino.rviz' configuration by default, to be able to visualize the 3D model of the robot and information form the topics.
```
Comment (jballoffet) [inline andino_bringup/README.md:24]: typo: "from the topics"

File: andino_bringup/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+## Launch Files
+
+### Main file
+
+The main launch file is the `andino_robot.launch.py` file. This file executes all necessary nodes for Andino Robot to work.
+
+It includes:
+
+- `andino_description.launch.py`: file to load the 3D model of the robot from the URDF.
+- `andino_control.launch.py`: file for the ROS2 control nodes and drivers.
```
Comment (jballoffet) [inline andino_bringup/README.md:15]: nit: "ROS 2 control" or `ros2_control`.

**Review by jballoffet — CHANGES_REQUESTED** [review]
Thanks @JesusSilvaUtrera! I left some minor comments, PTAL.

**Comment by JesusSilvaUtrera** [comment]
@jballoffet I have updated the README and the branch, if you think it is good we can merge it.

**Review by jballoffet — APPROVED** [review]
LGTM, thanks for contributing to Andino!

**Review by francocipollone — COMMENTED** [review]
Thanks!

**Pull request merged**